packages/node_modules/samples/browser-plugin-meetings/index.html

```diff
@@ -301,26 +311,26 @@ <h2>Then click:</h2>
           <button id="sendDtmf" title="Send DTMF" type="button">Send DTMF</button>
         </p>
         <p>
-          <label for="setMeetingQualityLevel">Set Meeting Quality Level</label>
-          <select id="setMeetingQualityLevel" name="setMeetingQualityLevel">
+          <label for="updateMeetingQualityLevel">Update Meeting Quality Level</label>
+          <select id="updateMeetingQualityLevel" name="updateMeetingQualityLevel">
             <option value="LOW">LOW</option>
             <option value="MEDIUM">MEDIUM</option>
             <option value="HIGH">HIGH</option>
           </select>
-          <button id="setMeetingQuality" title="meeting.setMeetingQuality()" type="button">meeting.setMeetingQuality()</button>
+          <button id="updateMeetingQuality" title="meeting.setMeetingQuality()" type="button">meeting.setMeetingQuality()</button>
         </p>
         <p>
-          <label for="setLocalVideoQualityLevel">Set Local Video Quality</label>
-          <select id="setLocalVideoQualityLevel" name="setLocalVideoQualityLevel">
+          <label for="updateLocalVideoQualityLevel">Update Local Video Quality</label>
+          <select id="updateLocalVideoQualityLevel" name="updateLocalVideoQualityLevel">
             <option value="LOW">LOW (320:180)</option>
             <option value="MEDIUM">MEDIUM (640:360)</option>
             <option value="HIGH">HIGH (1280:720)</option>
           </select>
           <button id="setLocalVideoQuality" title="meeting.setLocalVideoQuality()" type="button">meeting.setLocalVideoQuality()</button>
```

Comment:
This needs to match the app.js implementation

```suggestion
          <button id="updateLocalVideoQuality" title="meeting.setLocalVideoQuality()" type="button">meeting.setLocalVideoQuality()</button>
```